zerver/lib/user_topics.py

```diff
@@ -11,23 +11,64 @@
 from zerver.models import UserProfile, UserTopic, get_stream
 
 
-def get_topic_mutes(
-    user_profile: UserProfile, include_deactivated: bool = False
-) -> List[Tuple[str, str, float]]:
-    query = UserTopic.objects.filter(user_profile=user_profile, visibility_policy=UserTopic.MUTED)
-    # Exclude muted topics that are part of deactivated streams unless
+def get_user_topics(
+    user_profile: UserProfile,
+    include_stream_name: bool = False,
+    include_deactivated: bool = False,
+    visibility_policy: Optional[int] = None,
+) -> List[Dict[str, Any]]:
```

Comment:
Can you define an `UserTopicDict` TypedDict in `zerver/lib/types.py` for the format of the dictionaries this returns? We're trying to get away from any use of `Dict[str, Any]`, and it'll be helpful in this particular place to have a proper type.